tests/files/.gitignore

```diff
@@ -1,2 +1,3 @@
 *
-!.gitignore
\ No newline at end of file
+!.gitignore
+!database.sqlite
```

Comment:
Without the sample database we have to create a new database and create table in it, with or without dummy data. This would require additional dependency. I tried avoiding that by adding a simple 3KB sqlite db.